src/asynchronous.rs

```diff
@@ -153,7 +153,13 @@ pub(crate) async fn chown_to_non_root(dir: &Path) -> TmpPostgrustResult<()> {
         return Ok(());
     }
 
-    let (uid, gid) = &*POSTGRES_UID_GID;
+    let (uid, gid) = POSTGRES_UID_GID.get_or_init(|| {
+        User::from_name("postgres")
+            .ok()
+            .flatten()
+            .map(|u| (u.uid, u.gid))
+            .expect("no user `postgres` found is system")
```

Comment:
```suggestion
            .expect("no user `postgres` found in system")
```

Probably also want to factor this out into a new function